PublicPickView win 이 위로 lose 아래로 수정함

eundeang opened the pull request:
## Motivation ⍰

- figma에서 디자인 된것과는 다르게 구현 됨
- 질문들과 퍼센테이지 변수에 체계 X

<br>

## Key Changes 🔑

- win은 위쪽에 lose는 아래쪽으로 가도록 구현함
- 질문과 퍼센트를 튜플에 정리

<br>

## To Reviewers 🙏🏻

- [x] 코드 위쪽에 firstQuestion, secondQuestion 변경해서 한번만 확인 부탁드립니다

<br>

## Linked Issue 🔗

-

eundeang committed:
PublicPickView win 이 위로 lose 아래로 수정함

Minny27 commented:
음 개인적인 피드백입니다..!
1. pr올리기 전에 dev 브랜치 머지를 해서 conflict를 해결하고 commit -> push -> pr 올리면 더 좋을 거 같습니다!

2. firstAmount, secoundAmount 수치를 바꿨을 때 막대가 제대로 반영되지 않습니다.
<img src="https://user-images.githubusercontent.com/68800789/231769714-14e14f7c-f0f2-48a6-b5dd-e8379a6b24dc.png" width=20%>

3. 음 아직 전달할 view model이 모델링되진 않았지만 이전 화면에서 첫 번째, 두 번째 값을 넘겨줄 것이기 때문에 튜플보다는 Model로 타입을 정의하는 것이 좋아보입니다.
<img src="https://user-images.githubusercontent.com/68800789/231770169-115b5cf8-ef49-4992-92a2-bdc9efd49318.png" width=60%>

다른 부분은 다 잘 된 것 같고, 첫 번째 또는 두 번째 값이 랜덤으로 들어와서 win, lose를 유동적으로 가져가는 아이디어 좋은 것 같아용!

eundeang committed:
막대 변하지 않는 오류 수정

eundeang committed:
머지test

Minny27 committed:
Merge branch 'develop' into eundeang

Minny27 committed:
Update: 필요없는 파일 삭제
- FinalCircleTimer 삭제

# 제목의 길이는 최대 40글자까지 한글로 간단 명료하게 작성
# 제목을 작성하고 반드시 빈 줄 한 줄을 만들어야 함 # 제목에 .(마침표) 금지
# <label> 리스트
# Feat: 새로운 기능
# Fix: 버그 수정
# Add: Swift 파일 추가
# Update: Swift 파일 (삭제, 이동, 이름 변경)
# Style: 코드 스타일 혹은 포맷 등에 관한 커밋
# Docs: 문서 (md 파일 추가, 수정, 삭제)
# Refactor: 코드 가독성과 유지 보수의 편의에 관한 커밋
# Test: 테스트
# <description>
# 내용의 길이는 한 줄당 60글자 내외에서 줄 바꿈. 한글로 간단 명료하게 작성
# 어떻게 보다는 무엇을, 왜 변경했는지를 작성할 것 (필수)
# <issue-number>
# 연관된 이슈 첨부, 여러 개 추가 가능
